twentynineteen-blocks/style.css

```diff
@@ -112,6 +120,27 @@ Tags: one-column, flexible-header, accessibility-ready, custom-colors, custom-me
   }
 }
 
+/* Latest posts block */
+.wp-block-latest-posts li > a {
+	font-size: 2.25em;
+	font-weight: 700;
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+	color: #111;
+}
```

Comment:
I'm trying to match the h2 style here

Reply:
👍 Looks good. Just needed the line height, which I've added in 
f4da15b